src/components/DonateButton.jsx

```diff
@@ -184,7 +184,10 @@ class DonateButton extends React.Component {
 
     let { maxDonationAmount } = this.props;
     if (maxDonationAmount) {
-      if (dacId !== undefined && dacId !== 0) {
+      const amountDAC = parseFloat(maxDonationAmount - maxDonationAmount / 1.03)
```

Comment:
If you want to get the amount will be donated to dac, can simply write `maxDonationAmount * 0.03`